gulp/deployment.js

```diff
@@ -364,6 +364,7 @@ const replaceConfigurationValues = async function (folderPath) {
     .pipe(replace(/window\.localStorage\[\'evan-ens-events\'\]/g, `window.localStorage['evan-ens-events'] || ${JSON.stringify(config.bcConfig.nameResolver.domains.eventhub)}`))
     .pipe(replace(/window\.localStorage\[\'evan-ens-profiles\'\]/g, `window.localStorage['evan-ens-profiles'] || ${JSON.stringify(config.bcConfig.nameResolver.domains.profile)}`))
     .pipe(replace(/window\.localStorage\[\'evan-ens-mailbox\'\]/g, `window.localStorage['evan-ens-mailbox'] || ${JSON.stringify(config.bcConfig.nameResolver.domains.mailbox)}`))
+    .pipe(replace(/window\.localStorage\[\'evan-chain\'\]/g, `window.localStorage['evan-chain'] || ${runtime.environment}`))
```

Comment:
I would rather write a global available getChainEnvironment() function, bc like this it's not transparent for a developer, that `window.localStorage['evan-chain']` will be fallback to `runtime.environment`.

PS: the standardized way to access local storage would be using the getItem method. `localStorage.getItem('evan-chain');`

Reply:
ui-dapp-browser it self does not know at the beginning, which configuration should be used. During deployment, this configuration value will be replaced. We can prefill it with testcore, so the devs will not get confused.